pkg/block/device.go

```diff
@@ -139,6 +145,22 @@ func GetPartitionByUUID(s *sys.System, b Device, uuid string, attempts int) (*Pa
 	return nil, errors.New("no device found")
 }
 
+func lookupUUID(s *sys.System, path string) string {
+	out, err := s.Runner().Run("blkid", path)
+	if err != nil {
+		s.Logger().Error("blkid execution failed: %s", out)
```

Comment:
I think that `Error` here is a bit too much. It doesn't break anything but seeing ` time="2025-07-21T14:32:04Z" level=error msg="failed parsing blkid output"` is a bit annoying. `Warning` should be enough I think, @davidcassany WDYT?